build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        springBootVersion = '2.2.4.RELEASE'
+        springBootVersion = '2.2.5.RELEASE'
```

Comment:
Why not going directly to version 2.6.6 (which is the latest one according to [this](https://www.marcobehler.com/guides/spring-and-spring-boot-versions#:~:text=What%20is%20the%20latest%20Spring%20Boot%20version%3F,6.) ) ?

Reply:
I want minimal change with the least testing.
Considering the urgency and time required to qualify that version and test, this is a choice I made.

Reply:
In the next version we will update to 2.7.0 
https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#:~:text=1.-,Introduction,6.8%2C%206.9%2C%20or%207.

Reply:
https://plugins.gradle.org/plugin/org.springframework.boot

Reply:
Can we maybe create a ticket for bumping to latest version of springboot ? and maybe add a tool to the repo that will automatically create PR for bumping dependencies when they get released ? (I think dependabot can do something like that) what do you think ?

Reply:
https://scality.atlassian.net/browse/OSIS-46.
Yes, we also need to move to github actions and also automate this update.